cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -223,7 +223,7 @@ var InstanceTypes = map[string]*InstanceType{
 	"c5d.12xlarge": {
 		InstanceType: "c5d.12xlarge",
 		VCPU:         48,
-		MemoryMb:     131072,
```

Comment:
Em..Interesting. I do use the generator to populate the specs. Do you change manually or use the `gen.go`?

Reply:
Yeah, I was confused as well when I noticed it, even weirder I changed it by running `make generate` again, so I can only assume the API was returning different results when you last generated it.

Reply:
Yeah. The new values you give match with values in aws docs. Great!